tests/communicator/test_communicator.py

```diff
@@ -29,46 +30,40 @@ def close(*args, **kwargs):  # pylint: disable=unused-argument
     monkeypatch.setattr(traci, "close", close)
 
 
-class MockComponent(Component):
-    """Mock for a simple component to check if next tick is called"""
-
-    call_count = 0
-
-    def __init__(self):
-        Component.__init__(self, None, 1)
-
-    def next_tick(self, tick: int):
-        self.call_count += 1
-
-
-def test_simulation_runs(
-    mock_traci,
-):  # pylint: disable=unused-argument
+def test_simulation_runs(mock_traci):  # pylint: disable=unused-argument
     communicator = Communicator()
-    communicator.start()
-    sleep(0.1)
-    communicator.stop()
-    assert communicator.progress > 0
+    run_id = communicator.run()
+
+    while Communicator.state(run_id) != "PROGRESS":
```

Comment:
Where does this `"PROGRESS"` come from? Is that a celery thing?

Reply:
Yes! Absolutely 😇